src/Input/Input.js

```diff
@@ -6,7 +6,7 @@ import classNames from 'classnames';
 import { createStyleSheet } from 'jss-theme-reactor';
 import customPropTypes from '../utils/customPropTypes';
 
-function isDirty(obj) {
+export function isDirty(obj) {
```

Comment:
@mbrookes Thought about tat too) Shod function's name also be changed as it is related only to input/select elements?